L1Trigger/L1THGCal/python/hgcalLayersCalibrationCoefficients_cfi.py

```diff
@@ -0,0 +1,115 @@
+import FWCore.ParameterSet.Config as cms
+
+# coefficient evaluated from SinglePhoton pT=50 GeV considering all the layers in EE
+dEdX_weights = cms.vdouble(0.0,
```

Comment:
These are the weights derived with the matrix inversion method, which is not the same method as used to derive the dEdX weights used in the local reco, right? In that case it might be better to have a name that better reflects what these values are.